lib/net/imap.rb

```diff
@@ -443,6 +443,11 @@ def login(user, password)
     #
     # A Net::IMAP::NoResponseError is raised if the mailbox does not
     # exist or is for some reason non-selectable.
+    #
+    # If the server supports the UIDPLUS extension it may return an additional
```

Comment:
Adding a reference.  🙂
```suggestion
    # If the server supports the [UIDPLUS[https://www.rfc-editor.org/rfc/rfc4315.html]]
    # extension it may return an additional
```
n.b. this messes up the paragraph's line-breaks, but GitHub still doesn't allow multi-line suggestions. 😉